giraph-examples/src/main/java/org/apache/giraph/examples/DiffusionMigrationBlockFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.giraph.examples;
```

Comment:
Let's use some consistent code style to format the sources. I suggest Google's java formatting: http://google.github.io/styleguide/javaguide.html. You can use it directly in Eclispe/IDEA and apply formatting changes automatically.